### PR TITLE
Adds a README file

### PR DESCRIPTION
**Pull request by vibhav2799**
As title.

@AlpriElse

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+### LearnSeed
```
Comment (AlpriElse) [inline README.md]: Where's the space

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+An online course explorer.
```
Comment (AlpriElse) [inline README.md]: What is? WTF is an?

File: README.md (hunk 1 above)
Comment (vibhav2799) [reply README.md]: Nope, not having it.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+### LearnSeed
+
```
Comment (AlpriElse) [inline README.md]: Who made this repo?

Comment (vibhav2799) [reply README.md]: Don't fool around.

**Pull request merged**